cmd/analyze_deps.go

```diff
@@ -53,6 +62,20 @@ to quickly create a Cobra application.`,
 			printChain(chains[maxDepth+1])
 		}
 
+		// print all the cycles
+		if cyclesMode {
+			fmt.Println("All cycles in dependencies are: ")
+			cycles := getCycles(cycleChains)
+
+			for _, c := range cycles {
+				fmt.Println()
+				for _, d := range c {
+					fmt.Print(d + " -> ")
```

Comment:
can you perhaps use strings.join here?

Reply:
I didn't know of this, thanks! Have replaced it now.